types/math-expression-evaluator/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for math-expression-evaluator 1.2
+// Type definitions for math-expression-evaluator 1.3
 // Project: https://github.com/bugwheels94/math-expression-evaluator
 // Definitions by: DefinitelyTyped <https://github.com/DefinitelyTyped>
+//                 Adam Zerella <https://github.com/azerella>
```

Comment:
To confirm, are you looking to be an owner of this package and review PRs for it? https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

If you're just trying to note your contribution, you don't need to add this. But, I do see that there are no owners here (in which case we'd just remove the DT part and put your info there instead).